src/blas/backends/rocblas/rocblas_extensions.cpp

```diff
@@ -0,0 +1,290 @@
+               int64_t lda, uint8_t ao, cl::sycl::buffer<int8_t, 1> &b, int64_t ldb, int8_t bo,
+               float beta, cl::sycl::buffer<int32_t, 1> &c, int64_t ldc,
+               cl::sycl::buffer<int32_t, 1> &co) {
+    throw unimplemented("blas", "gemm_bias", "for column_major layout");
+}
+
+void gemm_bias(cl::sycl::queue &queue, transpose transa, transpose transb, offset offsetc,
+               int64_t m, int64_t n, int64_t k, float alpha, cl::sycl::buffer<uint8_t, 1> &a,
+               int64_t lda, uint8_t ao, cl::sycl::buffer<uint8_t, 1> &b, int64_t ldb, uint8_t bo,
+               float beta, cl::sycl::buffer<int32_t, 1> &c, int64_t ldc,
+               cl::sycl::buffer<int32_t, 1> &co) {
+    throw unimplemented("blas", "gemm_bias", "for column_major layout");
+}
+
+void gemmt(cl::sycl::queue &queue, uplo upper_lower, transpose transa, transpose transb, int64_t n,
+           int64_t k, float alpha, cl::sycl::buffer<float, 1> &a, int64_t lda,
+           cl::sycl::buffer<float, 1> &b, int64_t ldb, float beta, cl::sycl::buffer<float, 1> &c,
+           int64_t ldc) {
+    throw unimplemented("blas", "gemmt", "for column_major layout");
+}
+
+void gemmt(cl::sycl::queue &queue, uplo upper_lower, transpose transa, transpose transb, int64_t n,
+           int64_t k, double alpha, cl::sycl::buffer<double, 1> &a, int64_t lda,
+           cl::sycl::buffer<double, 1> &b, int64_t ldb, double beta, cl::sycl::buffer<double, 1> &c,
+           int64_t ldc) {
+    throw unimplemented("blas", "gemmt", "for column_major layout");
+}
+
+void gemmt(cl::sycl::queue &queue, uplo upper_lower, transpose transa, transpose transb, int64_t n,
+           int64_t k, std::complex<float> alpha, cl::sycl::buffer<std::complex<float>, 1> &a,
+           int64_t lda, cl::sycl::buffer<std::complex<float>, 1> &b, int64_t ldb,
+           std::complex<float> beta, cl::sycl::buffer<std::complex<float>, 1> &c, int64_t ldc) {
+    throw unimplemented("blas", "gemmt", "for column_major layout");
+}
+
+void gemmt(cl::sycl::queue &queue, uplo upper_lower, transpose transa, transpose transb, int64_t n,
+           int64_t k, std::complex<double> alpha, cl::sycl::buffer<std::complex<double>, 1> &a,
+           int64_t lda, cl::sycl::buffer<std::complex<double>, 1> &b, int64_t ldb,
+           std::complex<double> beta, cl::sycl::buffer<std::complex<double>, 1> &c, int64_t ldc) {
+    throw unimplemented("blas", "gemmt", "for column_major layout");
+}
+
+// USM APIs
+
+cl::sycl::event gemm_bias(cl::sycl::queue &queue, transpose transa, transpose transb,
+                          offset offsetc, int64_t m, int64_t n, int64_t k, float alpha,
+                          const int8_t *a, int64_t lda, int8_t ao, const int8_t *b, int64_t ldb,
+                          int8_t bo, float beta, int32_t *c, int64_t ldc, const int32_t *co,
+                          const cl::sycl::vector_class<cl::sycl::event> &dependencies) {
```

Comment:
Here and everywhere else:

```suggestion
                          const std::vector<cl::sycl::event> &dependencies) {
```